tempest/api/workloadmgr/trust/test_trust.py

```diff
@@ -0,0 +1,162 @@
+
+            #Execute scheduler-trust-validate CLI command
+            self._execute_scheduler_trust_validate_cli()
+
+            #Fetch trust list from API
+            trust_list = self.get_trusts()
+
+            #Verify if trust details returned in steps 3 and 4 match
+            found = False
+            for trust in trust_list:
+                if trust['name'] == self.wlm_trust['name']:
+                    found = True
+                    break
+            if found and self.wlm_trust_valid and self.wlm_scheduler:
+                reporting.add_test_step("Verify valid trust", tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Verify valid trust", tvaultconf.FAIL)
+                reporting.set_test_script_status(tvaultconf.FAIL)
+
+            #Delete the trust using API
+            if self.delete_trust(self.wlm_trust['name']):
+                reporting.add_test_step("Delete trust", tvaultconf.PASS)
+            else:
+                raise Exception("Delete trust")
+
+            #Execute scheduler-trust-validate CLI for workload WL-1
+            self._execute_scheduler_trust_validate_cli()
+
+            #Verify if trust details are returned appropriately
+            if not self.wlm_trust and not self.wlm_trust_valid and \
+                    self.wlm_scheduler:
+                reporting.add_test_step("Verify broken trust", tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Verify broken trust", tvaultconf.FAIL)
+                reporting.set_test_script_status(tvaultconf.FAIL)
+
+            #Create the trust again using API
+            trust_id = self.create_trust(tvaultconf.trustee_role)
+            if trust_id:
+                reporting.add_test_step("Create user trust on project", tvaultconf.PASS)
+            else:
+                raise Exception("Create user trust on project")
+
+            #Execute scheduler-trust-validate CLI for workload WL-1
+            self._execute_scheduler_trust_validate_cli()
+
+            #Verify if trust details are returned appropriately
+            trust_list = self.get_trusts()
+            found = False
+            for trust in trust_list:
+                if trust['name'] == self.wlm_trust['name']:
+                    found = True
+                    break
+            if found and self.wlm_trust_valid and self.wlm_scheduler:
+                reporting.add_test_step("Verify valid trust", tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Verify valid trust", tvaultconf.FAIL)
+                reporting.set_test_script_status(tvaultconf.FAIL)
+
+        except Exception as e:
+            reporting.add_test_step(str(e), tvaultconf.FAIL)
+            reporting.set_test_script_status(tvaultconf.FAIL)
+        finally:
+            reporting.test_case_to_write()
+
+
+    @decorators.attr(type='workloadmgr_api')
+    def test_3_cleanup(self):
```

Comment:
we can skip numbering cleanup and simply call it as "test_cleanup"

Reply:
Added numbering to ensure execution at the end only